index.html

```diff
@@ -115,6 +115,37 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">nilleh</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://twitter.com/nillehtwit" target="_blank">tweetr</a>
+          </p>
+          <p class="about">stranger on the internet who just so happens to be learning german.</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="First Resource">
+                  r/shitposting
+                </a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Second Resource">
+                  W3Schools
+                </a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Third resource">
+                  r/shitposting
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```

Comment:
Please change this comment to
`<!-- ________ Your Name Contributor card END ________ -->`